### DISCUSSION TITLE
No forum access

### DISCUSSION THREAD

**Discussion by arc1tec4** [bulletphysics/bullet3#4637] [General]
I was just curious as why I can't register for the forum.

Looking for assistance on integrating bullet into a language I'm assisting on, can provide more details if someone gets back to me also driving a tractor right now sorry.

Thanks.

**Comment by arc1tec4** [comment]
So it's just dead?

**Reply by tamaynard** [reply]
Yes https://github.com/bulletphysics/bullet3/pull/4583#issuecomment-2046192548

**Reply by arc1tec4** [reply]
Ok, thanks for the heads up.
